src/monster.cpp

```diff
@@ -932,7 +926,7 @@ void Monster::onThinkDefense(uint32_t interval)
 		}
 	}
 
-	if (!isSummon() && summons.size() < mType->maxSummons) {
+	if (!isSummon() && summons.size() < mType->maxSummons && hasFollowPath) {
```

Comment:
There isn't much to comment code-wise. I'm mostly concerned about the side effects of these changes. Have they been thoroughly tested? Otherwise I rather keep this PR open until after the 1.2 release.